deploy/example_catalog/cr-app-training-engine.json

```diff
@@ -53,26 +53,26 @@
             {
                 "cardinality": "1",
                 "configPackage": {
-                    "packageURL": "http://16.143.21.166/thirdparty/k8sapps/flask_configure/flask_configure.tgz"
+                    "packageURL": "file:///opt/configscript/flask_configure.tgz"
                 },
                 "id": "LoadBalancer",
-                "imageRepoTag": "docker.io/saurabhjogalekarhpe/flask-and-inf-eng"
+                "imageRepoTag": "docker.io/rsupekar/kd-flask_serving:latest"
             },
             {
                 "cardinality": "1+",
                 "configPackage": {
-                    "packageURL": "http://16.143.21.166/thirdparty/k8sapps/flask_configure/flask_configure.tgz"
+                    "packageURL": "file:///opt/configscript/flask_configure.tgz"
                 },
                 "id": "RESTServer",
-                "imageRepoTag": "docker.io/saurabhjogalekarhpe/flask-and-inf-eng"
+                "imageRepoTag": "docker.io/rsupekar/kd-flask_serving:latest"
             },
             {
                 "cardinality": "1",
                 "configPackage": {
-                    "packageURL": "http://16.143.21.166/thirdparty/k8sapps/notebook_sc/appconfig-1.8.tgz"
+                    "packageURL": "file:///opt/configscript/appconfig-1.8.tgz"
                 },
                 "id": "controller",
-                "imageRepoTag": "docker.io/saurabhjogalekarhpe/training-engine"
+                "imageRepoTag": "docker.io/rsupekar/kd-scikittraining_gpu:latest"
```

Comment:
We should put under bluedata rather than rsupekar on docker.io. Are you not able to put under bluedata, We might have to add you if you are unable to. Let me know.

Also can I see the diff in docker file

Reply:
I was not able to put under bluedata, denied access. I would like to be added to bluedata group on dockerhub.
Diff for flask: https://www.diffchecker.com/Fd5Itxwz
Diff for training: https://www.diffchecker.com/j07Akn6Z

Reply:
i added you, could you try

Reply:
what is flask? there should be one training with flask and one notebook right ?

Reply:
the first diff is for training with flask, second one is for notebook.

Reply:
why were these not updated to 2.0. Are we able to import these ?
tensorflow-data-validation
tensorflow-transform
tensorflow-model-analysis

Reply:
https://github.com/tensorflow/tfx
the compatibility of those specific package versions with tensorflow 2.0 can be found in the above link

Reply:
ok good if it works.

Reply:
Just change the docker image path and looks good to merge

Reply:
made the changes to download bluedata images